_posts/2024-11-28-openloco-v24.11.md

```diff
@@ -0,0 +1,80 @@
+---
+title:      "OpenLoco version 24.11"
+tagline:    "Release blog post"
+author:     "Duncan Frost"
+date:       2024-11-28 12:30:00 +0000
+layout:     single
+classes:    wide
+categories: news
+
+---
+
+OpenLoco v24.11 is out! In this release we've added our initial OpenGraphics objects,
+finished all of the 'paint' code, fixed a bunch of bugs.
+
+For a complete summary of changes, please find the
+[OpenLoco changelog](https://github.com/OpenLoco/OpenLoco/releases/tag/v24.11) on GitHub.
+
+## Assorted bug fixes (#2623, #2676, #2678, #2690, #2691, #2708, #2717, #2721, #2723, #2726, #2729, #2731, #2732, #2735, #2752, #2770)
```

Comment:
We really should find a way to link these, or just omit them, imo. To the average user, this long list of numbers won't mean anything.